image-layout.md

```diff
@@ -0,0 +1,45 @@
+## Open Container Initiative Image Layout Specification
+
+The OCI Image Layout is a slash separated layout of OCI content-addressable 
+This layout MAY be used in a variety of different transport mechanisms from archive formats (e.g. tar, zip) or used in shared filesystem environments (e.g. nfs) or in networked file fetching systems (e.g. http, ftp, rsync).
+Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appopriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+
+The image layout has two top level directories: 
+
+- "blobs" contains content-addressable blobs
+- "refs" contains descriptors pointing to an image manifest list
+
+This is an example image layout:
+
+```
+$ cd example.com/app/
+$ find .
+.
+./blobs
+./blobs/sha256:afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51
```

Comment:
I'd like to hear @jstarks response but checkout this [superuser question](http://superuser.com/questions/461320/get-a-colon-in-a-windows-filename):

> Your options are to strip the colons on the source, or __replace them with a unicode character that looks like a colon but isn't.__

Not sure if that is a viable option. 😄 

> And this page should link to a list somewhere that spells out the
algorithm behind each short form (e.g. ‘sha256’ → “SHA-256 in [1]”).

These probably belong in https://github.com/opencontainers/image-spec/blob/master/media-types.md or we can pull in the digest portion of the manifest specification.